packages/auth-local/client/components/RegisterForm/utils/joi.js

```diff
@@ -1,35 +1,63 @@
 import Joi from '@hapi/joi'
 
+// VALIDATION SETTINGS
+const defaultNameValidationSettings = Joi.string().required()
+  .regex(/^.+\s.+$/)
+  .messages({
+    'any.required': 'Fill in the field',
+    'string.empty': 'Fill in the field',
+    'string.pattern.base': 'Please enter your full name'
+  })
+
+const defaultEmailValidationSettings = Joi.string()
+  .email({ minDomainSegments: 2, tlds: { allow: false } })
+  .required()
+  .messages({
+    'any.required': 'Fill in the field',
+    'string.empty': 'Fill in the field',
+    'string.email': 'Invalid email'
+  })
+
+const defaulPasswordValidationSettings = Joi.string().required()
```

Comment:
`default`

please use a spell checker in your IDE. For example for VS Code:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker